code/game/machinery/doors/airlock.dm

```diff
@@ -956,11 +956,11 @@ var/list/airlock_overlays = list()
 		charge.ex_act(1)
 		detonated = 1
 		charge = null
-		for(var/mob/living/carbon/human/H in orange(1,src))
+		for(var/mob/living/carbon/human/H in orange(2,src))
 			H.Paralyse(8)
-			H.adjust_fire_stacks(1)
+			H.adjust_fire_stacks(20)
```

Comment:
Oh boy an instant kill for everyone near the door if they don't get medical help that sounds fun